docs/content/error/$compile/noctrl.ngdoc

```diff
@@ -0,0 +1,68 @@
+@ngdoc error
+@name $compile:noctrl
+@fullName Controller is required.
```

Comment:
If I am not mitaken, this error will also be thrown if there is `bindToController: true` but no `controler:...`. So the literals have to account for all cases (i.e. `bindToController: true` but no `controller` or no controller identifier.